### PR TITLE
Deployments Dashboard

### PR DESCRIPTION
**Pull request by noisekit**

### CODE REVIEW

File: e2e/dashboard.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+  return out.join('\n');
+}
+
+module.exports = {
+  dashboard,
+};
+
+if (require.main === module) {
+  require('./inspect');
+  dashboard().then(prettyMd).then(console.log);
```
Comment (0xjocke) [inline e2e/dashboard.js:133]: Dashboard return html string but we call prettyMd?

File: .circleci/config.yml [hunk 2]
```diff
@@ -460,6 +460,62 @@ jobs:
...
+      - store_artifacts:
+          path: "./e2e/deployments"
+
+      - run:
+          name: "Generate docs"
+          environment:
+            DEBUG: "e2e:*"
+          command: |
+            mkdir -p docs/for-developers
+            node e2e/dashboard > docs/for-developers/deployment-info.md
```
Comment (0xjocke) [inline .circleci/config.yml:490]: Same thing here, shouldn't this be `.html`

File: e2e/dashboard.js (hunk 1 above)
Comment (noisekit) [reply e2e/dashboard.js:133]: No it does not. It returns MD that has html in it for correct formatting.

File: .circleci/config.yml (hunk 2 above)
Comment (noisekit) [reply .circleci/config.yml:490]: it is md not html.

**Review by 0xjocke — APPROVED** [review]

**Pull request merged**